Dining/forms.py

```diff
@@ -97,10 +103,15 @@ def save(self):
         # also check if it has changed from earlier status
         old_list = DiningList.objects.get(id=self.instance.id)
 
-        if 0 < self.instance.dinner_cost_single != old_list.dinner_cost_single:
-            self.instance.dinner_cost_total = self.instance.dinner_cost_single * self.instance.diners
+        total_dinner_cost = self.cleaned_data['dinner_cost_total']
 
-        self.instance.save(update_fields=self.Meta.fields)
+        if total_dinner_cost > 0:
+            s_cost = total_dinner_cost / self.instance.diners.count()
+            # round up slightly, to remove missing cents
+            s_cost = round(s_cost+Decimal(0.0049), 2)
```

Comment:
I'd suggest something like: `s_cost = Decimal(s_cost).quantize(Decimal('.01'), rounding=ROUND_UP)`

Reply:
Dat is wel beter ja. Wist niet dat Decimal daar een unieke methode voor had.